pages/transaction/[hash].tsx

```diff
@@ -236,13 +240,18 @@ const TransactionInfo: FC<TransactionInfoProps> = ({ data, loaded }) => {
     )
   }, [data?.notes?.length, data?.spends?.length])
 
+  const badge =
```

Comment:
I think showing expired here would be good eventually. We don't sync expiration sequence right now though maybe can do that in a follow up